### PR TITLE
Added loading indicator

### PR DESCRIPTION
**Pull request by magmat88**

### CODE REVIEW

File: src/app/components/mainContent/mainContent.component.tsx [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
-                <div className="articles-container">
-                    {content?.articles.length >= 3 &&
-                        content.articles
-                            ?.slice(0, 3)
-                            .map((article: ContentItem) => (
-                                <ContentTile contentItem={article} contentLabel={ContentLabel.Article} />
-                            ))}
-                </div>
+    useEffect(() => {
+      console.log(content.pending);
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.tsx:15]: in general we should not use console.log in the app.

File: src/app/components/mainContent/mainContent.component.tsx [hunk 2]
```diff
@@ -1,42 +1,65 @@
...
-                        content.blogs
-                            ?.slice(0, 3)
-                            .map((blog: ContentItem) => (
-                                <ContentTile contentItem={blog} contentLabel={ContentLabel.Blog} />
-                            ))}
-                </div>
+    if (content.pending) {
+        return <LoadingIndicator />;
+    }
+    if (content?.articles?.length) {
```
Comment (earendillo) [inline src/app/components/mainContent/mainContent.component.tsx:21]: We need to refactor this component but let's do this as a work for separate ticket.

File: src/app/modules/content/content.state.ts [hunk 3]
```diff
@@ -2,38 +2,37 @@ import { CONTENT_STORE_KEY } from './content.module';
...
     imageUrl: string;
     newsSite: string;
-    summary: string;
     publishedAt: string;
+    summary: string;
+    title: string;
     updatedAt: string;
-}
-
-export type Report = ContentItem
```
Comment (earendillo) [inline src/app/modules/content/content.state.ts:14]: why did you removed extedning types for reports, blogs and articles? please revert

Comment (earendillo) [reply src/app/modules/content/content.state.ts:14]: OK I see it's moved down in the file

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/mainContent/mainContent.component.tsx (hunk 1 above)
Comment (magmat88) [reply src/app/components/mainContent/mainContent.component.tsx:15]: console.log removed

**Review by earendillo — APPROVED** [review]

**Pull request merged**